.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] `node_modules` folder should be in the `.gitignore` file, so git ignore it and wound be pushed to your Github repo, kinky consider adding node_modules to this file